src/components/src/main/java/org/apache/jmeter/assertions/JSONPathAssertion.java

```diff
@@ -137,15 +137,15 @@ private void doAssert(String jsonString) {
         }
 
         if (isExpectNull()) {
-            throw new IllegalStateException(String.format("Value expected to be null, but found '%s'", value));
+            throw new IllegalStateException(String.format("Value in json path '%s' expected to be null, but found '%s'", getJsonPath(), value));
```

Comment:
Should `value` be `objectToString(value)` like below?

Reply:
I accepted your changes.